npc/re/jobs/novice/academy.txt

```diff
@@ -13,80 +13,226 @@
-			mes "^4d4fffLeft mouse click allows basic game controls.";
-			mes "Such as conversation with NPC, movement, and attacks.^000000";
-			next;
-			cutin "fly_trock",2;
-			mes "[Captain Carocc]";
-			mes "Many youngsters visit the ^4d4fffRune Midgard Kingdom^000000 to chase their dreams.";
-			mes "Izlude is a satellite city of Prontera, the capital of the Rune Midgard Kingdom.";
-			next;
-			mes "[Captain Carocc]";
-			mes "Izlude always welcomes young adventurers like you who have just started their journey to the World.";
-			mes "You are one of them of course.";
+//== Starting Point  (iz_int) =================================
+iz_int,18,26,0	script	startpoint	HIDDEN_WARP_NPC,1,1,{
+OnTouch:
+	if (!izintspawn) {
+		specialeffect2 EF_ANGEL3; // On official it is some kind of Poring Angel, but I can't find it
+		navigateto("int_land", NAV_NONE, 1); // individual map name not required
+		izintspawn = 1;
+	}
+	end;
+}
+
+//== Wounded Swordsman (iz_int) =================================
+iz_int,56,32,3	script	Wounded Swordsman#izint	4_TOWER_01,8,8,{
+	if (!@display) {
+		classchange 4_TOWER_02;
+		npctalk "Ohhhhhhh....";
+		@display = 1;
+		end;
+	}
+	if(checkquest(21001) == -1) {
+		mes "[Wounded]";
+			mes "Wow! Thank you a lot!";
+			mes "I don't know how this happend to our ship";
+			mes "but we should go to see the captain.";
 			next;
-			mes "[Captain Carocc]";
-			mes "I would like to tell you more stories, but it's time to get ready for arrival.";
-			mes "There is an academy in Izlude for newcomers like yourself.";
+			mes "[Wounded]";
+			mes "... ohh, it seems my body is injured too much.";
+			mes "Maybe you can go without me?";
+			setquest(21001);
+			close;
+	}
+	else {
+		mes "[Wounded]";
+			mes "Luckily the water does not occupy the ship.";
+			mes "I wait here, you should go and see the captain.";
 			next;
+			mes "[Wounded]";
+			mes "I'm fine! I'm fine!";
+			mes "Don't worry about me, you should really go to the captain!";
+	}
+	cutin "tutorial02", 4;
+	mes "^4d4dff!- Information -!^000000";
+	mes "NPC Quest Received.";
+	mes "^4d4dffQuestinfo Shortcut is Alt + U^000000 You can check your quest status there anytime.";
+	close2;
+	cutin "", 255;
+	end;
+
+OnInit:
+	questinfo 21001, QTYPE_QUEST, 1, Job_Novice;
+	end;
+
+OnTouch_:
+	if (@cd + 15 > gettimetick(2))
+		end;
+	@cd = gettimetick(2);
+	npctalk "Urgh... please... help me...", "", bc_self;
+	sleep 7000;
+	npctalk "In order to talk with me.. click my body..";
+	end;
+}
+
+int_land,58,69,5	script	Sailor#int_land	4W_SAILOR,{
+	if (!checkquest(21002) || checkquest(21002) == 2) {
+		mes "[Sailor]";
+		mes "The ship is heading to Izlude soon~!";
+		mes "I'll let you know when we set sail~!";
+		close;
+	} else {
+		if (countitem(6008) >= 2) {
+			delitem 6008, countitem(6008);
+			completequest(21002);
+			mes "[Sailor]";
+			mes "Unbelievable, perfect! Any chance you want to join my crew?";
+			mes "Enough talking!!";
+			mes "Come on, we're ready to set sails!";
```

Comment:
Why use "read" instead of "ready"? If use "read" I think this sentence will be no meaning